internal/controller/constants.go

```diff
@@ -0,0 +1,33 @@
+package controller
+
+const (
+	/*** application server configuration file ***/
+	// OLSConfigName is the name of the OLSConfig configmap
+	OLSConfigCmName = "olsconfig"
+	// OLSNamespaceDefault is the default namespace for OLS
+	OLSNamespaceDefault = "openshift-lightspeed-operator"
```

Comment:
Do we want to fix to specific Namespace ? We can get the current namespace of the operator and look for the CR there . Whichever namespace the operator pod is running thats where we will look for the CR.

Reply:
Yes. The operator watch its own namespace for CR by default and creating the resources at the same namespace as the CR lives.
The env var `WATCH_NAMESPACE` is used to define which namespace the operator watches over. If it is empty, the operator watches over all namespaces.

Reply:
By our last Discussion the WATCH_NAMESPACE = <Namespace  where the operator runs> . So we should have no case where we will watch all namespaces . The deployment yaml for operator , will have this 
```        - name: WATCH_NAMESPACE
          valueFrom:
            fieldRef:
              apiVersion: v1
              fieldPath: metadata.namespace
```

Reply:
we should plan on the OLS operator being a cluster scoped operator (meaning it can only be installed on the cluster once), and also its operand should be a singleton (meaning that the operator only cares about/watches for a single CR instance:  an instance in a specific namespace(openshift-lightspeed) with a specific name (`cluster` is a common name used for singletons like this).

if users create other instances of the CRD the operator can ignore them, or it can update them w/ status information indicating they are being ignored.

Reply:
@bparees the olsconfig CRD will be always named `cluster` 
If I have a namespace `xavier` I should be able to install to there right ?

Reply:
Per discussion with @bparees we will expect the operator watch single namespace - `openshift-lightspeed-operator`